koku/masu/test/database/test_aws_report_db_accessor.py

```diff
@@ -328,17 +324,13 @@ def test_back_populate_ocp_infrastructure_costs(self, mock_unleash, mock_execute
 
         mock_jinja.reset_mock()
         mock_execute.reset_mock()
-        mock_unleash.reset_mock()
-        is_savingsplan_cost = False
-        mock_unleash.return_value = is_savingsplan_cost
         accessor.back_populate_ocp_infrastructure_costs(start_date, end_date, report_period_id)
 
         sql_params = {
             "schema": self.schema,
             "start_date": start_date,
             "end_date": end_date,
             "report_period_id": report_period_id,
-            "is_savingsplan_cost": is_savingsplan_cost,
         }
 
         accessor.prepare_query.assert_called_with(sql, sql_params)
```

Comment:
We can just remove this whole half of the test since the only thing that changed was the savings plan flag.